paperclip-permanent_records.gemspec

```diff
@@ -27,7 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'permanent_records', '>= 5.0'
```

Comment:
Do we need to have a look at any expected incoming changes from permanent_records? (IIRC the only reference would be in https://github.com/emjot/paperclip-permanent_records/blob/0.6.0/lib/paperclip/permanent_records/active_record.rb#L6 so I guess we should be fine?)

Reply:
I'd say so, too.